presto-main/src/main/java/com/facebook/presto/server/security/KerberosConfig.java

```diff
@@ -47,6 +51,7 @@ public KerberosConfig setServiceName(String serviceName)
         return this;
     }
 
+    @NotNull
```

Comment:
I'm not sure this is correct. There is a check for null in `SpnegoFilter` which makes setting the keytab optional. If we're going to require it, that check should be removed.